ui/apps/platform/src/Components/CompoundSearchFilter/types.ts

```diff
@@ -78,6 +88,9 @@ export type ImageSearchFilterConfig = {
 
 export type ImageAttribute = keyof ImageSearchFilterConfig['attributes'];
 
+export type ImageAttributeInputType =
+    ImageSearchFilterConfig['attributes'][keyof ImageSearchFilterConfig['attributes']]['inputType'];
+
```

Comment:
Would importing this from our type utils be equivalent?
```suggestion
export type ImageAttributeInputType = ValueOf<ImageSearchFilterConfig['attributes']>['inputType'];
```